recipes/qiskit-metal/yum_requirements.txt

```diff
@@ -0,0 +1,22 @@
+mesa-libGL-devel
+mesa-libGLU-devel
+mesa-libEGL-devel
+libX11-devel
+libXt-devel
+libXtst-devel
+libXrender-devel
+libXext-devel
+libXdmcp-devel
+libXcomposite-devel
+libXcursor-devel
+libxcb-devel
+libXau-devel
+libXi-devel
+libXScrnSaver-devel
+libXrandr-devel
+alsa-lib-devel
+xcb-util-devel
+xorg-x11-server-Xvfb
+pciutils-devel
+libXdamage
+libXfixes
```

Comment:
Are all of these just needed to import the package?

Reply:
Right, because it depends on PySide2 and imports it when you `import qiskit_metal` these were needed for the import test. I copied this list from [the PySide2 recipe](https://github.com/conda-forge/pyside2-feedstock/blob/5e658fbb0fa21a94684fb117c2da5016e47e726d/recipe/yum_requirements.txt).

Reply:
Ahh makes sense